examples/projection/projection.description

```diff
@@ -0,0 +1,9 @@
+//Projection involves  selecting only some of the attributes from the input
+// stream to be inserted into an output stream, renaming the attributes, add
+// constant values to the output stream and perform logical or mathematical operations.
+// In this example, events related to person being monitored and there will be a
+// children's day wish triggered when there is a person found whose age is less
+// than or equal to 16.
+// Ballerina Streams is an experimental feature. Please use `--experimental` flag when compiling Ballerina files
```

Comment:
// Ballerina Streams is an experimental feature. Please use the `--experimental` flag when compiling Ballerina files // that have streaming constructs.

Reply:
Updated.